inventories/netplan_cluster_example.yaml.j2

```diff
@@ -0,0 +1,35 @@
+# Copyright (C) 2024, SFL (https://savoirfairelinux.com)
+# SPDX-License-Identifier: Apache-2.0
+# This netplan files defines the path of the cluster network which is managed
+# by systemd. Especially, it attributes its cluster IP addr to each node.
+network:
+  version: 2
+  renderer: networkd
+  ethernets:
+    "{{ team0_0|default('0') }}":
+      mtu: 1800
```

Comment:
Why set the MTU to 1800?

Reply:
team0 need to have an increased MTU because of vxlan running over it.

Reply:
Done

Reply:
There is no vxlan with team0.
Recommended MTU for Ceph is 9000: https://docs.redhat.com/en/documentation/red_hat_ceph_storage/4/html/configuration_guide/ceph-network-configuration#verifying-and-configuring-the-mtu-value_conf

I suggest to use this value.